### DISCUSSION TITLE
검수 작업 관련

### DISCUSSION THREAD

**Discussion by cyannn1** [uotalkie/dol-kr#164] [General]
@mansus-traveler 
안녕하세요? 번역 검수 커밋 체크해주셔서 감사합니다, pr156에서 코멘트 남겨주신 내용도 확인했어요! 고생 많으셨습니다!
말씀하신 대로 pr156은 아직 건드리지 않은 상황이라 괜찮지만, 검수 작업을 하는 사람이 늘어났으니 다른 사람 작업 상황을 모르면 중복 작업을 들어갈 우려도 있으니, 현재 어떤 Pull request를 검수하고 있는지 진행도를 공유하면 좋을 것 같아서 discussion 에 글을 남깁니다! 
코멘트란에 작업 현황을 남기겠습니다! 차후 다른 PR을 검수하실 때 참고해주세요~

**Comment by cyannn1** [comment]
24.07.30 기준

pr/150 - 검수 완료, 커밋 (체크해주신 부분 수정하여 당일 중 반영 예정)
overworld-town\loc-street\events.twee (8326 - 9824) 


pr/151 - 검수 완료, 커밋
overworld-town/loc-dance-studio/main.twee 일부 번역
overworld-town/loc-home/ward.twee 일부 번역

pr/154 - 검수 진행중
overworld-town/loc-sea/events.twee 번역 - 검수 완료
overworld-town/loc-sea/main.twee 번역 - 검수 완료
overworld-town/loc-sea/widgets.twee 번역 - 검수 완료
overworld-town/special-eden/main.twee 번역 - 진행중 (942까지)


8월 1일까지 pr/154 검수 완료를 목표로...진행하고 있습니다!

**Comment by mansus-traveler** [comment]
감사합니다. 저도 일단 어디 검수하고 있는지 표시해 놓겠습니다.

pr/156 - 검수 진행중
base-system/pregnancy/children-events.twee 번역 - 진행중 (339까지)

**Comment by cyannn1** [comment]
2024.08.01
pr/154 검수 후 develop-cyannn1 브랜치에 커밋했습니다!
이번 주말까지 컴퓨터 사용 시간이 적어질 것 같아, 만약 다음 주 월요일에 pr/155 체크가 시작되지 않았다면 거기서부터 진행하겠습니다!
(pr/155 체크가 진행되고 있으면 pr/156 시작할게요~)

**Comment by mansus-traveler** [comment]
pr/156 검수를 끝까지 (~627 라인까지) 진행하긴 했으나, 중간에 파일이 한 번 수정되어 검수를 위해 남겨둔 코멘트 대부분이 날아가 버렸습니다. 이전에 남겼던 코멘트는 https://github.com/uotalkie/dol-kr/pull/156/files/668827a27d6c7db1b24970ed003dd494338efbf8 를 참조하시기 바랍니다.

그리고 여러가지 면에서 코멘트로 남기는 방식으로 검수를 진행하는 건 좀 불안해 보이네요. cyannn1님처럼 develop 브랜치 하나 파서 거기에서 수정한 다음 병합하는 게 더 좋은 방법일 듯한데 혹시 pull request 적용한 develop branch를 만드는 거 어떻게 하는 건지 알려주실 수 있을까요? 저 혼자 조금 끙끙대 봤는데 잘 안 되는 바람에...

**Reply by cyannn1** [reply]
안녕하세요! 수정하면 코멘트가 날아가는군요.......주의해야겠네요

pull request 적용한 develop branch를 만든다는게 정확히 어떤 뜻인지 모르겠네요.(제가 git 다뤄보는게 이번이 처음이라서 아직 기능이 낯서네요)
우선 저는 https://github.com/uotalkie/dol-kr/discussions/152 의 uotalkie 님이 달아주신 코멘트를 참고해서 진행하고 있습니다. 제가 진행한 순서는

1.github desktop에서 develop 브랜치를 베이스로 하는 신규 브랜치(develop-cyannn1)를 만들고

2.-(코멘트에 uotalkie님이 적어주신 내용대로)웹에서 가져오기를 원하는 pull request에 들어가 pull request merge-open this in Github Desktop를 눌러서 제 브랜치에 합치거나
-Recent branches 펼치면 나오는 Pull request 목록에서 원하는 pr을 눌러서 리스트에 가져온 후, Choose a branch to merge into~ 를 눌러서 제 브랜치에 merge 하고 

3.검수 후 commit하는 순서로 진행하고 있습니다!

말씀하시는게 현재 리스트에 있는 모든 pull request가 적용된 develop 브랜치를 만드시겠다는 건가요?
pull request를 한 번에 하나씩 적용하는 것은 가능한데, 한꺼번에 모든 pull request를 merge하는게 가능한지는 모르겠네요.

**Reply by mansus-traveler** [reply]
아닙니다, 설명해주신 내용이 제가 원하던 정보가 맞습니다. 알려주셔서 감사합니다!

**Comment by cyannn1** [comment]
0807 - pr/155 시작하겠습니다!
